host-go/runtimes/js/runtime.go

```diff
@@ -0,0 +1,186 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//go:build js
+
+package js
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"sync"
+	"syscall/js"
+
+	"github.com/lens-vm/lens/host-go/engine/module"
+	"github.com/lens-vm/lens/host-go/engine/pipes"
+)
+
+type wRuntime struct {
+	webAssembly js.Value
+}
+
+var _ module.Runtime = (*wRuntime)(nil)
+
+func New() module.Runtime {
+	// https://developer.mozilla.org/en-US/docs/WebAssembly/JavaScript_interface
```

Comment:
praise: Thank you for including the links to the documentation, I might try and do this in my work.

suggestion: It might be nice to include a brief summary along side the link - it is more immediate to readers, and if the link dies or is moved we will still have some documentation.  (same goes for all doc link locations)